### ISSUE TITLE
Topic Review

### ISSUE THREAD

**Issue by FrankSylvia** [center-for-knowledge-communication/mathspring#201]
It would be helpful if MathSpring had a function that groups students for teachers that it thinks are in similar spots

**Comment by IvonArroyo** [comment]
Agreed! I have been thinking of this for a while, creating clusters of
students.

------
Ivon Arroyo              https://advancedlearningtech.com/
Associate Professor
College of Education and College of Computer Science
A259 LGRC (CICS) and Furcolo W261 (College of Ed)
University of Massachusetts Amherst
Amherst, MA 01002

**Comment by IvonArroyo** [comment]
We need a spec for this. It should be probably a new report. It should show the values of the criteria that they are being clustered on: topic mastery and for which topics, effort, emotion, a lot of the information that is shown in all the other reports.

**Comment by perspegrity5** [comment]
Great idea, ant this would be a good thing to ask Susan and some teachers about.  There seem to be so many ways students could be clustered automatically.
In addition, students could have an open ended  keyworld field allowing teachers to create their own clusters. E.G. "fallingBehind"  "needsFractionsRemediation"
